aicsimageio/__init__.py

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from .aics_image import AICSImage  # noqa: F401
+from .aics_image import imread  # noqa: F401
+from .aics_image import imread_dask  # noqa: F401
+
+# Do not edit this string manually, always use bumpversion
+# Details in CONTRIBUTING.md
+__version__ = "3.3.0"
```

Comment:
4.0.0 ;)

Reply:
Nah. This waits for bumpversion. This was just from merge from master.